pyproject.toml

```diff
@@ -2,16 +2,11 @@
 # See LICENSE file for licensing details.
 
 [tool.poetry]
-name = "mongodb-k8s"
-version = "0.0.1-dev.0"
-description = ""
-license = "Apache-2.0"
-authors = [
-    "Mykola Marzhan <mykola.marzhan@canonical.com>"
-]
-readme = "README.md"
-homepage = "https://charmhub.io/mongodb-k8s?channel=6/edge"
-repository = "https://github.com/canonical/mongodb-k8s-operator"
```

Comment:
I think we need to move this information to the new [project](https://python-poetry.org/blog/announcing-poetry-2.0.0#supporting-the-project-section-in-pyprojecttoml-pep-621) section so we don't lose this information :)

(See the other suggestion comment)